app/scripts/components/errors/report.js

```diff
@@ -28,23 +28,53 @@ var ErrorReport = React.createClass({
     else return truncate(string);
   },
 
+  renderSingleError: function (report, trigger) {
+    if (!trigger) {
+      trigger = this.truncate(report);
+    }
+    // No need to make error collapsible if the truncated
+    // output is the same length as the original ouptut
+    if (typeof report === 'string' &&
+        report === trigger &&
+        report.length === trigger.length) {
+      return <p>{report}</p>;
+    }
+    return (
+      <Collapsible trigger={trigger} triggerWhenOpen={'Show less'}>
+        {report}
+      </Collapsible>
+    );
+  },
+
   renderReport: function (report) {
     if (typeof report === 'string') {
-      return <p key={report}><strong>Error:</strong> <Collapsible trigger={this.truncate(report)} triggerWhenOpen={report}/></p>;
+      return (
+        <div key={report}>
+          <strong>Error:</strong>
+          { this.renderSingleError(report) }
+        </div>
+      );
     } else if (report instanceof Error) {
       let name = report.name || 'Error';
       let message, stack;
       if (!report.message) {
         message = JSON.stringify(report);
       } else {
         message = report.message;
-        stack = report.stack ? report.stack.split('\\n').map(s => <p key={s}>{s}</p>) : null;
+        stack = report.stack
+          ? report.stack.split(`\n`).map((s, index) => <p key={index}>{s}</p>)
+          : null;
       }
-      return <div><p><strong key={message}>{name}: </strong> <Collapsible trigger={this.truncate(message)} triggerWhenOpen={message}/></p><Collapsible trigger={this.truncate(stack)} triggerWhenOpen={stack}/></div>;
```

Comment:
I tested this out on several pages (dashboard homepage, collections list, etc) and it seemed like having the error message and error stack was quite redundant, because the error message was always included in the stack, but I welcome any feedback here